src/EnergyPlus/DataLoopNode.hh

```diff
@@ -145,9 +145,13 @@ namespace DataLoopNode {
     enum class ConnectionObjectType
     {
         Invalid = -1,
+        Undefined,
         AirConditionerVariableRefrigerantFlow,
         AirLoopHVAC,
+        AirLoopHVACDedicatedOutdoorAirSystem,
+        AirLoopHVACExhaustSystem,
```

Comment:
`ConnectionObjectType` enum was missing some components that do CompSets but don't manage any nodes directly.